GRDB/Core/Database+Schema.swift

```diff
@@ -365,8 +365,12 @@ extension Database {
                 throw DatabaseError(message: "no such table: \(tableName)")
             }
         }
+        // The "hidden" column magic numbers come from the SQLite source code.
+        // The values 2 and 3 refer to virtual and stored generated columns,
+        // respectively. Search for COLFLAG_VIRTUAL in
+        // https://www.sqlite.org/cgi/src/file?name=src/pragma.c&ci=fca8dc8b578f215a
         let columns = try ColumnInfo
-            .fetchAll(self, sql: "PRAGMA table_info(\(tableName.quotedDatabaseIdentifier))")
+            .fetchAll(self, sql: "SELECT * FROM pragma_table_xinfo('\(tableName)') WHERE hidden IN (0,2,3)")
```

Comment:
The `table_xinfo` pragma was introduced in SQLite [3.26.0](https://sqlite.org/releaselog/3_26_0.html), and GRDB can target older versions of SQLite (iOS 10.0 ships SQLite 3.14.0, for example).

So we need to run `table_info` or `table_xinfo` depending on the SQLite version.

I recommend performing a runtime check (`sqlite3_libversion_number() < 3026000`) instead of compiler checks (`#if GRDBCUSTOMSQLITE`).